src/app/core/components/sidenav/sidenav.component.ts

```diff
@@ -26,6 +27,7 @@ export class SidenavComponent extends WithUnsubscribe() implements OnInit, OnDes
 
   public routes: Array<SidenavRoute> = cloneDeep(sidenavRoutes);
   public nonDraggableRoutes = nonDraggableRoutes;
+  public alwaysVisibleRouteId: SidenavRouteId = 'VMS';
```

Comment:
I think that It's better to create a method for that. Something like canBeHidden or isAlwaysVisible